tensorrt_llm/llmapi/llm_args.py

```diff
@@ -1027,18 +979,18 @@ def _maybe_update_config_for_consistency(
         # max_beam_width is not included since vague behavior due to lacking the support for dynamic beam width during
         # generation
         black_list = set(["max_beam_width"])
-        executor_config_attrs = set(attr for attr in dir(ExecutorConfig)
-                                    if not attr.startswith('_')
-                                    and callable(getattr(ExecutorConfig, attr)))
+        executor_config_attrs = set(
+            attr for attr in dir(_ExecutorConfig) if not attr.startswith('_')
```

Comment:
Might this also be something that could be handled with a `PrivateAttrt` that gets initialized under the hood? Though I suppose the goal here is to make sure that all `_ExecutorConfig` attributes are available at the top-level config?

Reply:
Sorry, I am not fully following. Could you help elaborate with some sample code?

Reply:
I was thinking that you might be able to define the fields you're pulling from the executor config as a `PrivateAttr` in here so that you can hide them from being serialized. But that does mean you need to maintain that mapping whenever the `ExecutorConfig` is updated.

Reply:
After checking the implementation of `_maybe_update_config_for_consistency`, I found it is stateless, and it doesn't change any members of BaseLlmArgs. So it seems unnecessary to declare PrivateAttr for this function.